repos/fdbt-site/src/pages/api/deleteProductGroup.ts

```diff
@@ -0,0 +1,71 @@
+import { NextApiResponse } from 'next';
+import {
+    deleteProductGroupByNocCodeAndId,
+    // getAllProductsByNoc,
+    // getProductGroupByNocAndId,
+} from '../../data/auroradb';
+import { redirectToError, redirectTo, getAndValidateNoc } from '../../utils/apiUtils/index';
+import { NextApiRequestWithSession } from '../../interfaces';
+//import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+// import { getProductsMatchingJson } from '../../data/s3';
+// import { updateSessionAttribute } from '../../utils/sessions';
+// import { VIEW_PRODUCT_GROUP } from '../../constants/attributes';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    try {
+        const { query } = req;
+        const id = Number(query?.id);
+
+        if (!Number.isInteger(id)) {
+            throw new Error('insufficient data provided for delete query');
+        }
+        const nocCode = getAndValidateNoc(req, res);
+
+        // TODO: Code to be added after finalizing the structure of product group in capped ticket
+        // const products = await getAllProductsByNoc(nocCode);
+        // const matchingJsonLinks = products.map((product) => product.matchingJsonLink);
+
+        // const tickets = await Promise.all(
+        //     matchingJsonLinks.map(async (link) => {
+        //         return await getProductsMatchingJson(link);
+        //     }),
+        // );
+
+        // const ticketsWithProductGroup = tickets.filter(
```

Comment:
For now, lets just delete the group and we can figure out the plan later. No commented out code is needed.